ci/Makefile

```diff
@@ -24,28 +24,30 @@ ifeq ($(IN_HAIL_CI),1)
 
 PROJECT := $(shell gcloud config get-value project)
 
-CI_UTILS_LATEST = gcr.io/$(PROJECT)/ci-utils:latest
-CI_UTILS_IMAGE = gcr.io/$(PROJECT)/ci-utils:$(shell docker images -q --no-trunc ci-utils | sed -e 's,[^:]*:,,')
+BUILDER_UTILS_LATEST = gcr.io/$(PROJECT)/builder:latest
+BUILDER_UTILS_IMAGE = gcr.io/$(PROJECT)/builder:$(shell docker images -q --no-trunc builder | sed -e 's,[^:]*:,,')
```

Comment:
I feel like this isn't needed. You already define these in the builder makefile, exactly

https://github.com/hail-is/hail/blob/ee5f23ae2ca8ac69cc328ce8f0d707f944764bff/ci/Makefile#L27

Reply:
I couldn't figure out how to get the variable values back to the ci makefile so they could be used in the jinja template further down.

Reply:
Ah